src/mds/objects.py

```diff
@@ -114,6 +114,12 @@ async def create_object(
             f"Invalid authz.resource_paths, must be valid list of resources, got: {authz.get('resource_paths')}",
         )
 
+    # alias should not be 'upload'. This will help avoid conflicts between
+    # POST /api/v1/objects/{GUID or ALIAS} and POST /api/v1/objects/upload endpoints
+    logger.debug("checking for allowable aliases")
+    if any(alias == "upload" for alias in aliases):
```

Comment:
since we are doing this check (`value != "upload"`) 5 times now, i think it might be worth creating a global `FORBIDDEN_IDS = ["upload"]` to reuse in `objects.py` and `maintain.py` so it'll be easier to update it everywhere at once if we ever need to forbid a new ID. But i realize we've gone back and forth a few times on this so you can make the call

Reply:
Yes, good idea. Much cleaner to have the list in one place - placed in config for now.

Reply:
i'd rather set it in one of the 2 files and import it in the other than make it configurable since it's an internal requirement 🤔

Reply:
Oh, right! Moving into objects.